atlasdb-config/src/test/java/com/palantir/atlasdb/factory/TransactionManagersTest.java

```diff
@@ -922,7 +922,7 @@ private void verifyFeedbackIsReportedToService() {
                         refreshableRuntimeConfig,
                         USER_AGENT,
                         DialogueClients.create(Refreshable.only(ServicesConfigBlock.builder().build()))
-                                .withBlockingExecutor(PTExecutors.newCachedThreadPool("atlas-dialogue-blocking")));
+                                .withBlockingExecutor(PTExecutors.newCachedThreadPool()));
```

Comment:
Sorry I wasn't clear, I meant that we can remove this line entirely, but it doesn't hurt anything to leave around either.